packages/Rakefile

```
@@ -19,12 +19,10 @@ class GroongaPackageTask < PackagesGroongaOrgPackageTask
   end
 
   def detect_target_apache_arrow_version
-    cmake_yml = File.join(__dir__,
-                          "..",
-                          ".github",
-                          "workflows",
-                          "cmake.yml")
-    File.read(cmake_yml)[/ref: apache-arrow-([\d.]+)/, 1]
+    release_url = "https://api.github.com/repos/apache/arrow/releases/latest"
+    release_info = JSON.parse(URI.open(release_url).read)
```

How about putting it together as below?

```suggestion
    latest_tag_name = JSON.parse(URI.open(release_url).read)["tag_name"]
```

---

fix: [17280a1](https://github.com/groonga/groonga/pull/2208/commits/17280a126ef0ba29061b72ace05a7ae048eba528)